File: windows-ui/GSMRelayDesktop/Models/AppNotification.cs
namespace GSMRelayDesktop.Models;

public class AppNotification
{
    public long Id { get; set; }
    public string Message { get; set; } = "";
    public string Type { get; set; } = "info";
    public long Timestamp { get; set; }
    public bool Read { get; set; }
}

File: android/app/src/main/java/com/security/gsmrelay/model/AppNotification.kt
package com.security.gsmrelay.model

data class AppNotification(
    val id: Long,
    val message: String,
    val type: String = "info",
    val timestamp: Long,
    val read: Boolean = false,
    val relayPhone: String = "",
    val relayName: String = ""
)
